install.sh

```diff
@@ -7,6 +7,21 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
+# Check if script was executed with root/sudo
+if [ "$(id -u)" -ne 0 ]; then
+    if [ "$CI" != "true" ]; then
+        echo "You should be using sudo, are you sure you want to continue? [y/n]"
```

Comment:
I would prefer to use "It's recommended to use.." rather than "You should be using.."

```suggestion
        echo "It is recommended to run the script as the root user (by using sudo or as a direct root user) to avoid permission issues. Are you sure you want to continue? [y/n]"
```